src/vsphere_cpi/lib/ruby_vim_sdk/soap/stub_adapter.rb

```diff
@@ -32,9 +33,9 @@ def invoke_method(managed_object, method_info, arguments, outer_stub = nil)
           elsif status == 200
             result = object
           elsif object.kind_of?(Vmodl::MethodFault)
-            raise SoapError.new(object.msg, object)
+            raise VimSdk::SoapError.new(object.msg, object)
```

Comment:
Is it necessary to use `VimSdk::SoapError` instead of just `SoapError` as we are already inside `VimSdk` module.

Reply:
Yes its needed.